types/fuzzyset/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for fuzzset 1.0
+// Project: https://github.com/axiak/fuzzyset
+// Definitions by: Louis Grignon <https://github.com/lgrignon>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface FuzzySet {
+    get(candidate: string): Array<[number, string]>;
+    add(value: string): boolean;
+    length(): number;
+    isEmpty(): boolean;
+    values(): string[];
+}
+
+declare function FuzzySet(source: string[], useLevenshtein?: boolean, gramSizeLower?: number, gramSizeUpper?: number): FuzzySet;
```

Comment:
Note that the one on NPM at least seems to be available as a module instead of as a global. So it should have `export = FuzzySet;` and be imported with `import FuzzySet = require("fuzzyset");` if so.